Tests/Unit/Content/ContentResolverTest.php

```diff
@@ -39,9 +40,12 @@ protected function setUp(): void
         $this->mediaSelectionResolver = $this->prophesize(ContentTypeResolverInterface::class);
 
         $this->contentResolver = new ContentResolver(
-            [
-                'media_selection' => $this->mediaSelectionResolver->reveal(),
-            ]
+            new RewindableGenerator(
```

Comment:
you probably do not need to change this when using `iterable` as type of the argument above

Reply:
It is not really possible without